content/Cookies/Cookies.facts.md

```diff
@@ -0,0 +1,56 @@
+### Facts for Cookies lesson:
+
+A cookie is a small amount of information stored on the user's computer for application state tracking and identification purposes.
+
+The `setcookie()` function creates and deletes cookies in PHP. It sends a URL encoded cookie together with the rest of the HTTP headers.
+
+It has the following parameters:
```

Comment:
Will it be okay if we do not include/mention the `path`, `domain`, and `secure` parameters? We don't have an example that uses them. Our learners might wonder what they are for and question why it's not included in the lesson. Thoughts?

Reply:
Sorry, but I think this should be part of our facts since they are part of the parameters. For me, it is still necessary to introduce/include this in the facts even if we do not have some of this in our q. Please check. Thank you.

Reply:
noted, thanks.